api/v2/llm/suggestions.py

```diff
@@ -3,36 +3,38 @@
 from ninja import Router
 from ninja.errors import HttpError
 
-from api.v2.llm.schema import ModifySuggestionSchema
+from api.v2.llm.schema import ModifySuggestionSchema, RateSuggestionSchema
 from common.ai_review.dto import SuggestionState
 from common.models import Comment, SuggestedComment
 from common.utils import is_teacher
 
 router = Router()
 
 
+# TODO: Add check, to ensure that only teacher, that teach the course can accept/reject suggestions
+
+
 @router.post(
-    "/rate/{suggestion_id}",
+    "/{suggestion_id}/rate",
     url_name="rate_submit_suggestion",
     summary="Rate suggested comment",
     description="Rate an AI-suggested comment as helpful or not.",
 )
 @user_passes_test(is_teacher)
-def rate_submit_suggestion(request, suggestion_id: int, query: dict):
+def rate_submit_suggestion(request, suggestion_id: int, body: RateSuggestionSchema):
     suggestion = get_object_or_404(SuggestedComment, id=suggestion_id)
 
-    rating = query.get("rating")
-    if rating is None or not (0 <= rating <= 10):
+    if body.rating is None or not (0 <= body.rating <= 10):
```

Comment:
`body.rating is None` should be impossible, as it is checked already by Django Ninja in the schema, right?